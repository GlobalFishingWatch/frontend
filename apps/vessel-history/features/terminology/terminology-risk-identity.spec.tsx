import { Trans, useTranslation } from 'react-i18next'
import { render } from '@testing-library/react'

import TerminologyRiskIdentity from './terminology-risk-identity'

jest.mock('react-i18next')

describe('TerminologyRiskIdentity', () => {
  const mockUseTranslation: jest.Mock = useTranslation as jest.Mock
  const mockTrans: jest.Mock = Trans as jest.Mock
  beforeEach(() => {
    mockUseTranslation.mockReturnValue({
      t: (_ns: any, fallback: any) => fallback,
    })
    mockTrans.mockImplementation(({ i18nKey, children }) => children)
  })

  afterEach(() => {
    jest.clearAllMocks()
  })
  it('should render successfully', () => {
    const { baseElement } = render(<TerminologyRiskIdentity />)
    expect(baseElement).toMatchSnapshot()
  })
})
