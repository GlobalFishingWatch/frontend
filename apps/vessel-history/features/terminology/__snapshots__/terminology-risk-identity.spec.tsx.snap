// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TerminologyRiskIdentity should render successfully 1`] = `
<body>
  <div>
    <p>
      Identity refers to identifiers, based on available information, of who the vessel is. It includes characteristics such as type, size and authorizations, and other information about the individuals who own or operate them. Vessel identifiers are based on an amalgamation of key sources, including registry data, AIS transmissions, and other key sources. Identity also includes information related to aspects of the vessel identity, such as the vessel flag's presence on the Paris and/or Tokyo MOU list.
    </p>
  </div>
</body>
`;
